### DISCUSSION TITLE
Windows Version 3.10.0 is being reported as containing a malicious object.

### DISCUSSION THREAD

**Discussion by steveroberts1969** [massCodeIO/massCode#438] [Q&A]
I've been using versions up to 3.8.0 OK and decided to download the latest version for Windows 3.10.0 today. However, Kaspersky has flagged the download as containing a malicious object and subsequently deletes the file?

Kaspersky details as below:

Result:     Deleted: UDS:Trojan-Downloader.Win32.Bitser.fat
Object:     C:\Users\stever\Downloads\massCode.Setup.3.10.0.exe
Hash:     6d4baf4a56412acecd9fd6ee18d70d12bc315ef2156db0441296539e6f6af1fe

**Comment by antonreshetov** [comment]
That's certainly news...well, where did the viruses come from? 😄 If you downloaded from the release page, there can't be any viruses.

**Reply by steveroberts1969** [reply]
Thanks for the prompt reply, the file was downloaded from the link on your site: https://github.com/massCodeIO/massCode/releases/download/v3.10.0/massCode.Setup.3.10.0.exe
Kaspersky quarantines the file as soon as it has completed downloading. Have you had any other reports?
It is possible its a false positive (incorrectly being flagged). I will try scanning the file on another system that uses Bitdefender and let you know the results.

**Reply by antonreshetov** [reply]
I'm a Mac user )

**Reply by steveroberts1969** [reply]
Does that make your software Immune to vulnerabilities? Thanks for your reassuring comments. All I was doing was notifying you of a potential issue. I think I'll give this version a miss.

**Reply by antonreshetov** [reply]
There are few viruses on macOS. You can skip this version, it's up to you, but there's nowhere to get viruses. Logic.
